lib/src/allow_anything_parser.dart

```diff
@@ -36,6 +36,7 @@ class AllowAnythingParser implements ArgParser {
       bool negatable = true,
       void Function(bool)? callback,
       bool hide = false,
+      bool hideNegatable = false,
```

Comment:
nit: It isn't obvious what this does to me just from the name (I would guess it would entirely hide the option, not just remove the `[--no]` text). Maybe, `hideNegatedUsage`?